docs/DesignDocs/ExtendedEnclaveInitializationData.md

```diff
@@ -59,27 +57,27 @@ The additional union member is of type `oe_eeid_t`, which contains the following
 #ifdef EXPERIMENTAL_EEID
 typedef struct oe_eeid_t_
 {
+    uint32_t version;        /* version number of the structure */
     uint32_t hash_state[10]; /* internal state of the hash computation at the end of the enclave base image */
-    uint8_t *signature_size; /* size of signature */
-    uint8_t *signature;      /* base-image signature and associated data (for SGX, the complete sigstruct of the base image) */
+    uint64_t signature_size; /* size of signature */
     oe_enclave_size_settings_t size_settings; /* heap, stack and thread configuration for this instance */
     uint64_t vaddr;          /* location of the added data pages in enclave memory; EEID follows immediately thereafter */
-    uint64_t entry_point;    /* entry point of the image */
+    uint64_t entry_point;    /* entry point of the image, for SGX, matches TCS.OENTRY */
     uint64_t data_size;      /* size of application EEID */
```

Comment:
You changed the comment on data[] but now the comment on data_size no longer matches it